### PR TITLE
Seed Script, Husky, Lint Staged -- Improved Developer Experience

### PR DESCRIPTION
**Pull request by kerimhudson**
You're not going to like me for this one, but this integrates husky, prettier, and lint staged into our project to ensure our code is kept consistent and clean, and free from linting bugs!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,8 +20,25 @@
...
+    "prettier": "^2.3.2",
+    "pretty-quick": "^3.1.1",
     "prisma": "2.27.0",
+    "slugify": "^1.6.0",
     "tailwindcss": "^2.2.4"
+  },
+  "lint-staged": {
+    "**/*.js": "eslint --fix"
+  },
+  "husky": {
```
Comment (kerimhudson) [inline package.json:39]: Husky is a tool that allows us to run scripts before committing our staged changes.

File: package.json [hunk 2]
```diff
@@ -20,8 +20,25 @@
...
+    "husky": "^7.0.1",
+    "lint-staged": "^11.0.1",
     "postcss": "^8.3.5",
+    "prettier": "^2.3.2",
+    "pretty-quick": "^3.1.1",
     "prisma": "2.27.0",
+    "slugify": "^1.6.0",
     "tailwindcss": "^2.2.4"
+  },
+  "lint-staged": {
```
Comment (kerimhudson) [inline package.json:36]: Lint-staged is a tool that allows us to check only the staged files, so we don't have to format/check the whole codebase each time.

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["next", "prettier"]
```
Comment (kerimhudson) [inline .eslintrc:2]: ESLint is a tool to allow us to lint our codebase. Luckily, Next.js provides us with a configuration out of the box!

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "javascript.format.enable": true,
```
Comment (kerimhudson) [inline .vscode/settings.json:2]: This is to ensure that our code is formatted using prettier if we're using VSCode.

**Review by oksygenn — APPROVED** [review]

File: seed.js [hunk 5]
```diff
@@ -1,42 +1,142 @@
...
+        await prisma.content.create({
+            data: {
+                title: title,
+                description: faker.random.words(40),
+                image: faker.image.abstract(),
+                slug: slugify(title),
+                type: faker.datatype.number() > 4 ? "ARTICLE" : "VIDEO",
+                issues: {
+                    create: {
+                        issueId: issues[Math.floor(issues.length * Math.random())].id,
```
Comment (kerimhudson) [inline seed.js:131]: Assign a random category from the above categories array.

**Pull request merged**